params/params.go

```diff
@@ -0,0 +1,57 @@
+package params
+
+import (
+	"github.com/jfrog/jfrog-client-go/artifactory"
+	"reflect"
+)
+
+type GoResolverDeployer struct {
```

Comment:
I suggest renamkng *GoResolverDeployer* to *ResolverDeployer*.

Reply:
Done